### ISSUE TITLE
Python syntax warning: "is" with a literal. Did you mean "=="?

### ISSUE THREAD

**Issue by christoph-conrads** [code-saturne/code_saturne#78]
Code from [`bin/cs_mei_to_c.py`, line 2344](https://github.com/code-saturne/code_saturne/blob/d49b10cd1e45e1ff8205b8c4fb12cfa04ce5bd2d/bin/cs_mei_to_c.py#L2344):
```python
                if zone.getNature()['initialization'] is 'on':
```
Warning on Ubuntu 20.04 with Python 3.8.5:
```
/tmp/prefix/lib/python2.7/site-packages/code_saturne/cs_mei_to_c.py:2344: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if zone.getNature()['initialization'] is 'on':
```
The warning looks justified to me.

**Comment by YvanFournier** [comment]
Hello,

Yes, this was cleaned up in later versions. Since v6.0 is in "bug fix"/porting mode only, we may or may not change this in a future release.

**Comment by YvanFournier** [comment]
Fixed in commit 31781289b (v6.0 branch, will be n v6.0.7).
